### ISSUE TITLE
Error when trying to bind pipeline

### ISSUE THREAD

**Issue by therealnv6** [therealnv6/vx-gfx#9]
```
[2023-05-10 03:19:27.620] [error] validation layer: Validation Error: [ UNASSIGNED-CoreValidation-Shader-InputNotProduced ] Object 0: handle = 0xb991fa0000000024, type = VK_OBJECT_TYPE_SHADER_MODULE; | MessageID = 0x23e43bb7 | vkCreateGraphicsPipelines(): pCreateInfos[0] Vertex shader consumes input at location 0 but not provided
[2023-05-10 03:19:27.620] [error] validation layer: Validation Error: [ UNASSIGNED-CoreValidation-Shader-InputNotProduced ] Object 0: handle = 0xb991fa0000000024, type = VK_OBJECT_TYPE_SHADER_MODULE; | MessageID = 0x23e43bb7 | vkCreateGraphicsPipelines(): pCreateInfos[0] Vertex shader consumes input at location 1 but not provided
```

**Comment by dashwastakenx** [comment]
provide  the shader locations

**Comment by therealnv6** [comment]
Fixed in https://github.com/therealnv6/vx-gfx/commit/1c76c37b722eac12cce686a41c44da29c8783bcc

TL;DR pipeline was being constructed before I bound the descriptors to the pipeline.
